query.txt

```diff
@@ -0,0 +1,18 @@
+You are now connected to database "nc_games_test" as user "yasirmaj".
```

Comment:
Same as before. If this is a test file for just you, .gitignore it so that it doesn't become a part of the repo.